azure/iac/master-compliance-test.json

```diff
@@ -2555,7 +2555,7 @@
                             "id": "PR-AZR-0122-ARM",
                             "eval": "data.rule.region",
                             "message": "data.rule.region_err",
-                            "remediationDescription": "",
+                            "remediationDescription": "Make sure StorageAccount has (location='northeurope' or location='westeurope')",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.